ctapipe/analysis/Sensitivity.py

```diff
@@ -0,0 +1,790 @@
+from itertools import chain
+import astropy.units as u
+from astropy.table import Table
+from scipy.optimize import minimize
+import scipy.integrate as integrate
+
+import numpy as np
+
+from matplotlib import pyplot as plt
+
+
+# tau, the superior circle number
+np.tau = 2*np.pi
+
+__all__ = ["SensitivityPointSource"]
+
+
+def convert_astropy_array(arr, unit=None):
+    """
+    converts a python list of quantities into a quantified numpy array of the same
+    dimension
+
+    Parameters
+    ----------
+    arr : python list
+        list of quantities of same dimension (not strictly exact same unit)
+    unit : astropy unit, optional (default: None)
+        if set, uses this as the unit of the numpy array; if not, uses the unit of the
+        first element
+        ought to be of same dimension of the quantities in the list (there is no test)
+
+    Returns
+    -------
+    a : quantified numpy array
+    """
+
+    if unit is None:
+        unit = arr[0].unit
+    return np.array([a.to(unit).value for a in arr])*unit
+
+
+def crab_source_rate(energy):
+    '''
+    function for a pseudo-Crab point-source rate Crab source rate:
+        dN/dE = 3e-7  * (E/TeV)**-2.48 / (TeV * m² * s)
+    (watch out: unbroken power law... not really true)
+    norm and spectral index reverse engineered from HESS plot...
+
+    Parameters
+    ----------
+    energy : astropy quantity
+        energy for which the rate is desired
+
+    Returns
+    -------
+    flux : astropy quantity
+        differential flux at E
+
+    '''
+    #return 3e-7 * (energy/u.TeV)**-2.52 / (u.TeV * u.m**2 * u.s)
+    return 3e-7 * (energy/u.TeV)**-2.48 / (u.TeV * u.m**2 * u.s)
+
+
+def CR_background_rate(energy):
```

Comment:
We have this: http://docs.gammapy.org/en/latest/api/gammapy.spectrum.cosmic_ray_flux.html
I didn't check how this compares to your CR background rate.

In any case, ideally there should be a reference to a paper or webpage or something where this spectrum comes from ...